### PR TITLE
Registration: Gloria FX

### PR DESCRIPTION
**Pull request by kate-dzi**

### CODE REVIEW

File: applications/gloria.fx/squad.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Gloria FX
+
+https://gloriafx.com/
+
+`44` [**Бєлiзной** Андрiй](/players/beliznoy.andrei.19950604.jpg) `/1995-06-04/`
```
Comment (anton-kochev) [inline applications/gloria.fx/squad.md:9]: A newline is missing between lines.
Should be:

\+ `44` [**Бєлiзной** Андрiй](/players/beliznoy.andrei.19950604.jpg) `/1995-06-04/`
\+
\+ `4` [**Боговенко** Олег](/players/bohovenko.oleh.19891111.jpg) `/1989-11-11/`

File: applications/gloria.fx/squad.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Gloria FX
+
+https://gloriafx.com/
```
Comment (anton-kochev) [inline applications/gloria.fx/squad.md:3]: Please add also this link https://www.facebook.com/gloriafx and DOU (if you have)

**Review by anton-kochev — COMMENTED** [review]
Please provide another photo for `players/dzihovskyi.andrij.19880324.jpg`. The face is not well recognizable

File: applications/gloria.fx/squad.md (hunk 2 above)
Comment (kate-dzi) [reply applications/gloria.fx/squad.md:3]: Should I add links to the same line separated by commas?

Comment (anton-kochev) [reply applications/gloria.fx/squad.md:3]: Each link on the separate line, please

**Review by anton-kochev — COMMENTED** [review]
See comments above

**Review by anton-kochev — CHANGES_REQUESTED** [review]
Please fix all issues

**Comment by anton-kochev** [comment]
@steamhammer Documentation approved. Players check to be performed

**Comment by steamhammer** [comment]
> @steamhammer Documentation approved. Players check to be performed

All good

**Review by steamhammer — APPROVED** [review]
all good

**Review by anton-kochev — APPROVED** [review]

**Pull request merged**